spec/yaml/swiftnav/sbp/tracking.yaml

```diff
@@ -250,27 +250,19 @@ definitions:
       Tracking channel state for a specific satellite signal and
       measured signal power.
     fields:
-        - state:
-            type: u8
-            desc: Status of tracking channel
-            fields:
-                - 2-7:
-                    desc: Reserved
-                - 0-1:
-                    desc: Tracking mode
-                    values:
-                        - 0: Disabled
-                        - 1: Running
         - sid:
-            type: GnssSignal
+            type: GnssSignal16
             desc: GNSS signal being tracked
+        - fcn:
+            type: u8
+            desc: Frequency channel number
         - cn0:
-            type: float
-            units: dB Hz
-            desc: Carrier-to-noise density
+            type: u8
+            units: dB Hz / 4
```

Comment:
I copied this over from the obs, but shouldn't the unit be `dB Hz * 4`?

@cbeighley @mbavaro